### PR TITLE
chore: add empty message

### LINKED ISSUE #437: Missing Empty Recipients List Message in Staging
So the Recipients List is missing an empty message when a particular recipient can't be found.

This opens up a feature allowing the user to add a recipient if it can't be found in the list.
I think for version 1.0 We can show the empty message.
Let me know what you think.

Screenshot (63)

### PR DESCRIPTION
**Pull request by PeterKwesiAnsah**
This PR adds an empty message to the recipient list.
Fixes #437

### CODE REVIEW

File: packages/ui/primitives/document-flow/add-fields.tsx [hunk 1]
```diff
@@ -340,7 +340,9 @@ export const AddFieldsFormPartial = ({
               <PopoverContent className="p-0" align="start">
                 <Command>
                   <CommandInput />
-                  <CommandEmpty />
+                  <CommandEmpty>
+                    <span>No Recipient matching this description was found.</span>
```
Comment (dguyen) [inline packages/ui/primitives/document-flow/add-fields.tsx:344]: Hi Peter!

Could you please add the following className to the span:
- `text-muted-foreground inline-block px-4`

And also lowercase Recipient?

Thank you!

**Review by dguyen — CHANGES_REQUESTED** [review]

File: packages/ui/primitives/document-flow/add-fields.tsx (hunk 1 above)
Comment (PeterKwesiAnsah) [reply packages/ui/primitives/document-flow/add-fields.tsx:344]: alright @dguyen 
Thanks

Comment (PeterKwesiAnsah) [reply packages/ui/primitives/document-flow/add-fields.tsx]: @dguyen  updated

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks for your work!

giphy

**Pull request merged**